Oracle/PCORNetLoader_ora.sql

```diff
@@ -1276,11 +1276,11 @@ to_char(m.start_date,'HH:MI')  SPECIMEN_TIME,
 m.end_date RESULT_DATE,
 to_char(m.end_date,'HH:MI') RESULT_TIME,
 --CASE WHEN m.ValType_Cd='T' THEN NVL(nullif(m.TVal_Char,''),'NI') ELSE 'NI' END RESULT_QUAL, -- TODO: Should be a standardized value
-'NI' RESULT_QUAL, -- Temporary fix for KUMC
+'NI' RESULT_QUAL, -- Local fix for KUMC (temp)

```

Comment:
I'm uneasy about "Local fix for KUMC" comments. Of course it's a local fix: it's on our fork. But perhaps it serves to distinguish changes that we expect to upstream from those we don't.